ci/backup-metadata/install-cf-for-k8s.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```

Comment:
Certainly some overlap with our tasks to deploy cf-for-k8s, but might require some rework to work for both of us

Reply:
Yep, I don't think we should duplicate these. It should live in the capi pipeline https://capi.ci.cf-app.com/teams/main/pipelines/capi?group=capi-k8s-release